mmv1/third_party/terraform/services/container/resource_container_cluster.go.tmpl

```diff
@@ -2250,6 +2250,7 @@ func ResourceContainerCluster() *schema.Resource {
 			"user_managed_keys_config": {
 				Type:        schema.TypeList,
 				Optional:    true,
+				ForceNew:    true,
```

Comment:
What does this do?

Reply:
Marks the field as immutable so that changes trigger recreation: https://googlecloudplatform.github.io/magic-modules/best-practices/immutable-fields/